odoo/addons/test_impex/models.py

```diff
@@ -5,7 +5,7 @@
 
 
 def selection_fn(model):
-    return [(str(key), val) for key, val in enumerate(["Corge", "Grault", "Wheee", "Moog"])]
+    return [(str(key), val) for key, val in enumerate(["Foo", "Bar", "Qux", "Baz"])]
```

Comment:
Why?  I think that those values were chosen _on purpose_ to not be "foo", "bar", "baz", etc.